Angular/src/app/career-table/career-table.component.html

```diff
@@ -27,4 +31,4 @@
     </ng-container>
 </table>
 
-<br />
+<br />
```

Comment:
I'd use CSS (padding/margin) rather than a `<br/>` tag

Reply:
didn't need it anyway